nf_core/lint.py

```diff
@@ -451,6 +486,26 @@ def check_nextflow_config(self):
             else:
                 self.passed.append((4, "Config variable process.container looks correct: '{}'".format(container_name)))
 
+    def check_actions_branch_protection(self):
+        """Checks that the github actions workflows are valid.
+
+        Makes sure that ``nf-core lint`` runs in github actions workflows and that 
+        tests run with the required nextflow version.
+        """
+        wf = '.github/workflows/branch.yml'
+        fn = os.path.join(self.path, wf)
```

Comment:
As a general rule of thumb, when writing python I try to always avoid hard coding `/` directory separators. This is to at least try to give some support for windows users. The `os.path.join` function handles this in an agnostic manner.

```python
fn = os.path.join(self.path, '.github', 'workflows', 'branch.yml')
```

Reply:
good point, thanks! I tend to forget windows users...